src/server/debug.rs

```diff
@@ -682,8 +687,13 @@ impl<ER: RaftEngine> Debugger<ER> {
             })?;
 
             let applied_index = old_raft_apply_state.applied_index;
+            let commit_index = old_raft_apply_state.commit_index;
             let last_index = old_raft_local_state.last_index;
 
+            if last_index == applied_index && commit_index == applied_index {
```

Comment:
Why the change?

Reply:
Because if so the API will not change any state but will print too many unnecessary logs.

Reply:
Oh, sorry, I misread. Why check for `commit_index == applied_index`?

Reply:
Because the code below will modify `commit_index` to be equal `applied_index`